tests/congruence_tests/test_collections.py

```diff
@@ -42,3 +45,93 @@ def test_get_collection():
     assert (
         local_collection_info.config.params.vectors == remote_collection_info.config.params.vectors
     )
+
+
+def test_update_collection():
+    fixture_records = generate_fixtures()
+
+    remote_client = init_remote()
+    remote_collections = remote_client.get_collections()
+    for collection in remote_collections.collections:
+        remote_client.delete_collection(collection.name)
+
+    local_client = init_local()
+    init_client(local_client, fixture_records)
+
+    remote_client = init_remote()
+    init_client(remote_client, fixture_records)
+
+    local_client.update_collection(
+        collection_name=COLLECTION_NAME,
+        vectors_config = {
+            "text": models.VectorParamsDiff(
+                hnsw_config=models.HnswConfigDiff(
+                    m=32,
+                    ef_construct=123,
+                ),
+                quantization_config=models.ProductQuantization(
+                    product=models.ProductQuantizationConfig(
+                        compression=models.CompressionRatio.X32,
+                        always_ram=True,
+                    ),
+                ),
+                on_disk=True,
+            ),
+        },
+        hnsw_config=models.HnswConfigDiff(
+            ef_construct=123,
+        ),
+        quantization_config=models.ScalarQuantization(
+            scalar=models.ScalarQuantizationConfig(
+                type=models.ScalarType.INT8,
+                quantile=0.8,
+                always_ram=False,
+            ),
+        ),
+    )
+    remote_client.update_collection(
+        collection_name=COLLECTION_NAME,
+        vectors_config = {
+            "text": models.VectorParamsDiff(
+                hnsw_config=models.HnswConfigDiff(
+                    m=32,
+                    ef_construct=123,
+                ),
+                quantization_config=models.ProductQuantization(
+                    product=models.ProductQuantizationConfig(
+                        compression=models.CompressionRatio.X32,
+                        always_ram=True,
+                    ),
+                ),
+                on_disk=True,
+            ),
+        },
+        hnsw_config=models.HnswConfigDiff(
+            ef_construct=123,
+        ),
+        quantization_config=models.ScalarQuantization(
+            scalar=models.ScalarQuantizationConfig(
+                type=models.ScalarType.INT8,
+                quantile=0.8,
+                always_ram=False,
+            ),
+        ),
+    )
+
+    local_collection_info = local_client.get_collection(COLLECTION_NAME)
+    remote_collection_info = remote_client.get_collection(COLLECTION_NAME)
+
+    assert local_collection_info.config.params.vectors["text"].hnsw_config is None
+    assert local_collection_info.config.params.vectors["text"].quantization_config is None
+    assert local_collection_info.config.hnsw_config.ef_construct == 123
```

Comment:
In the local client `ef_construct` is set to 123 in an `update_collection` request, but it remains 100 which was the default.

Is this a problem or intended behavior for the local client?

The remote client works as expected.

Reply:
collection params doesn't have an effect in local client